src/content/guides/creating-clusters-via-crs-on-aws/index.md

```diff
@@ -223,6 +223,27 @@ As a result of rendering the CRs ([sample](/reference/kubectl-gs/template-cluste
 The resources can then be created by applying the manifest files to the Control Plane, e.g. `kubectl create -f <cluster manifest file>.yaml`.
 Of course, that requires the user to be authorized towards Kubernetes Control Plane API.
 
+## How to configure OIDC authentication at cluster creation using Cluster API
+
+Starting from version {{% first_aws_nodepools_version %}} on AWS, Giant Swarm has enabled the possibility to configure the OIDC parameters per cluster at creation stage. This feature includes an OIDC section in the `AWSCluster` CR with the necessary fields.
+
+In order to configure a cluster with OIDC, you will have to add these fields to the AWSCluster CR as following:
```

Comment:
This here says `AWSCluster`. The YAML below says `Cluster`. Something is off here.

Reply:
Also, what is https://pkg.go.dev/github.com/giantswarm/apiextensions@v0.4.20/pkg/apis/infrastructure/v1alpha2#AWSClusterSpecClusterOIDC for if this is supposed with the annotations?

Reply:
Nice catch, forgot to update yaml https://github.com/giantswarm/docs/pull/724